lib/npm/test.js

```diff
@@ -55,8 +55,14 @@ function test(context, next) {
       npmBin = path.join(path.dirname(npmBin), 'node_modules', 'npm', 'bin',
           'npm-cli.js');
     }
-    const proc = spawn(nodeBin, [npmBin, 'test', '--loglevel=' +
-        context.options.npmLevel], options);
+
+    let args = [npmBin, 'test', '--loglevel=' + context.options.npmLevel];
+
+    if (context.options.customTest) {
+      args = context.options.customTest;
+    }
```

Comment:
So you _could_ make this a ternary, which _might_ be clearer.

```js
    let args = [npmBin, 'test', '--loglevel=' + context.options.npmLevel];
 
    if (context.options.customTest) {
      args = context.options.customTest;
    }
```

```js
    const args context.options.customTest ?
      context.options.customTest :
      [npmBin, 'test', '--loglevel=' + context.options.npmLevel];
```

However I'm not actually sure that it's any clearer, so don't change it unless someone else prefers it.

The main reason I like it is because args can be `const`, so you know it's not changed later.